packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md

```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
+|[Toast alerts](#toast-alerts) |Use to present a system notification that should not interrupt the user’s current workflow. |Enter from the right edge of the screen and overlay content. |Remain until they time out, or the user dismisses them.
+|[Modal alerts](#modal-alerts) |Use when you must interrupt the task at hand until the user takes action. |Overlay the UI and prevent further user actions until the user closes the modal. |Remain until the user dismisses the modal.
 
 ### Inline alerts
-Inline alerts are synchronous and appear as the result of a user action or upon loading a page. They insert an alert at the top of the content area, push down other content, and will persist until the user closes them or navigates away from the page.
+Inline alerts are synchronous and appear as the result of a user action or upon loading a page. They insert themselves into the content area, push down other content, and will persist until the user closes them or navigates away from the page.
 
 ![inline notification placement](./img/inline-notification.png)
 
-An inline alert should never block the user from interacting with other content on a page. For notifications with a severity of info or warning it should be possible to dismiss an inline alert by clicking the Close button on the alert. If Close is not included, the alert will dismiss only when the condition raising it has been cleared.
+#### When to use
+Use an inline alert when you want to return feedback as the result of a user action within a page while allowing the user to continue working. Common uses for inline alerts include form validation or warning messages that should not interrupt the user’s flow.
 
-In addition to static message content and links, interactive elements may optionally be included in the body of an inline alert when the user is being asked for some input or feedback.
+An inline alert should never block the user from interacting with other content on a page. For informational alerts, the user should be able to dismiss the alert by clicking the Close button on the alert. For error conditions, we recommend that the alert is dismissed only when the condition raising it has been cleared.
 
-![interactive inline alert](./img/inline-interactive-alert.png)
+In addition to static message content and links, interactive elements may optionally be included in the body of an inline alert to ask the user for further input or feedback.
 
-#### When to use
-Use an inline alert when you want to return feedback as the result of a user action within a page while allowing the user to continue working. Common uses for inline alerts include form validation or warning messages that should not interrupt the user’s flow.
+<img src="./img/inline-interactive-alert.png" alt="interactive inline alert" width="798"/>
```

Comment:
@mcarrano I think we should be using the inline alert with actions styling like this for this example! Instead of the radio buttons... or I can add radio buttons into an inline alert if necessary
<img width="364" alt="Screen Shot 2020-06-03 at 3 38 14 PM" src="https://user-images.githubusercontent.com/30870144/83681322-3f1e5100-a5b0-11ea-8968-8a6c383df2f9.png">